scripts/start

```diff
@@ -1,23 +1,27 @@
 #!/bin/bash
 
-cd $(dirname $0)/..
-
-exists() {
-    [ -e "$1" ]
-}
-
-if exists Artemis-modules-extra; then
```

Comment:
this was important too - we don't want to scale sqlmap if the user is nto using sqlmap